external-crates/move/crates/move-analyzer/src/symbols.rs

```diff
@@ -388,7 +388,9 @@ impl fmt::Display for DefInfo {
                     f,
                     "{}fun {}::{}{}({}){}",
                     visibility_to_ide_string(visibility),
-                    mod_ident,
+                    // this mod_ident conversion will ensure that only pkg name (without numerical
```

Comment:
These two are sneaked in here after I noticed that the IDE sometimes displays package name as, for example `somepkg=0x42`, instead of simply `somepkg` (the conversion function, also used to create uniform map key from module ident normalizes this).